src/org-manager/containers/pending-organisations/pending-organisations.component.html

```diff
@@ -39,14 +40,15 @@ <h1 class="govuk-heading-xl">Organisations pending activation</h1>
               <td class="govuk-table__cell"><span class="hmcts-badge">{{org.status}}</span>
               </td>
               <td class="govuk-table__cell">
-                  <a class="govuk-link" [routerLink]="org.routerLink">View</a>
+                  <a class="govuk-link" [routerLink]="'/organisation-details/pendingOrganisations/' + org.organisationId">View</a>
```

Comment:
Better to use array notation for the link here:

`[routerLink]="['/organisation-details', 'pendingOrganisations', org.organisationId]"`